recipes/stages/segmentation/train.py

```diff
@@ -23,15 +32,15 @@
 params_config = dict(
     type='FreezeLayers',
     by_epoch=True,
-    iters=40,
+    iters=0,
     open_layers=[r'\w*[.]?backbone\.aggregator\.', r'\w*[.]?neck\.',
                  r'\w*[.]?decode_head\.', r'\w*[.]?auxiliary_head\.']
 )
 
 custom_hooks = [
     dict(type='LazyEarlyStoppingHook',
-         patience=5,
-         iteration_patience=1000,
+         patience=8,
+         iteration_patience=0,
          metric='mIoU',
```

Comment:
@chuneuny-emily then, I think this metric also should be changed with 'mDice', am I right?

Reply:
Yes. I changed it too.